### PR TITLE
Add some debug properties to BackdropFilterLayer

### LINKED ISSUE #101102: Debug properties of BackdropFilterLayer missed
When debugDumpLayerTree, we found that BackdropFilterLayer's properties are missed.
```
flutter:  │     │ ├─child 6: ClipRectLayer#b154d
flutter:  │     │ │ │ engine layer: ClipRectEngineLayer#6c8d6
flutter:  │     │ │ │ handles: 2
flutter:  │     │ │ │ clipRect: Rect.fromLTRB(39.2, 135.0, 67.2, 162.0)
flutter:  │     │ │ │ clipBehavior: Clip.hardEdge
flutter:  │     │ │ │
flutter:  │     │ │ └─child 1: BackdropFilterLayer#2376b
flutter:  │     │ │   │ engine layer: BackdropFilterEngineLayer#75b16
flutter:  │     │ │   │ handles: 2
flutter:  │     │ │   │
flutter:  │     │ │   └─child 1: PictureLayer#788c9
flutter:  │     │ │       handles: 1
flutter:  │     │ │       paint bounds: Rect.fromLTRB(39.2, 135.0, 67.2, 162.0)
flutter:  │     │ │       picture: Picture#81dd7
flutter:  │     │ │       raster cache hints: isComplex = false, willChange = false
```

### PR DESCRIPTION
**Pull request by wangying3426**
Fix https://github.com/flutter/flutter/issues/101102

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I signed the [CLA].
- [x] I listed at least one issue that this PR fixes in the description above.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by TahaTesser** [comment]
This PR requires test

File: packages/flutter/lib/src/rendering/layer.dart [hunk 1]
```diff
@@ -1956,6 +1956,13 @@ class BackdropFilterLayer extends ContainerLayer {
     addChildrenToScene(builder);
     builder.pop();
   }
+
+  @override
+  void debugFillProperties(DiagnosticPropertiesBuilder properties) {
+    super.debugFillProperties(properties);
+    properties.add(DiagnosticsProperty<ui.ImageFilter>('filter', filter));
+    properties.add(DiagnosticsProperty<BlendMode>('blendMode', blendMode));
```
Comment (goderbauer) [inline packages/flutter/lib/src/rendering/layer.dart:1964]: There is an `EnumProperty` which may be more appropriate for this one?

**Review by goderbauer — APPROVED** [review]
LGTM

**Comment by wangying3426** [comment]
> Please add all properties and this PR requires a test.

cc @TahaTesser Could you have look at this again? It needs two flutter members to review.

**Comment by goderbauer** [comment]
@Piinks Could you do a secondary review?

**Review by TahaTesser — APPROVED** [review]
LGTM

**Comment by TahaTesser** [comment]
@wangying3426 
Can you rebase or add an empty commit so Google Testing passes to get this merged?

**Review by Piinks — APPROVED** [review]
LGTM! Thank you!

**Comment by wangying3426** [comment]
> @wangying3426 Can you rebase or add an empty commit so Google Testing passes to get this merged?

The Google testing seems has some problem, and failed many times.

**Pull request merged**